internal/template-bundle/bundle_test.go

```diff
@@ -21,13 +23,19 @@ var _ = Describe("Template bundle", Ordered, func() {
 	var (
 		testBundle Bundle
 		nameSuffix string
+		testDir    string
 	)
 
 	BeforeAll(func() {
 		var err error
 		testBundle, err = ReadBundle("template-bundle-test.yaml")
 		Expect(err).ToNot(HaveOccurred())
 		nameSuffix = retrieveSuffix()
+		_, filename, _, ok := runtime.Caller(0)
+		Expect(ok).To(BeTrue())
+		fileAbsolutePath, err := filepath.Abs(filename)
+		Expect(err).NotTo(HaveOccurred())
+		testDir = filepath.Dir(fileAbsolutePath)
```

Comment:
Instead of reading files from the repo, can you create them in a temp directory when the test starts?
Ginkgo can create the directory for you:
```
tmpDir := GinkgoT().TempDir()
```

Reply:
Nice idea, thanks for pointing that out